packages/cozy-stack-client/src/SettingsCollection.js

```diff
@@ -13,10 +14,26 @@ class SettingsCollection extends DocumentCollection {
   /**
    * async get - Calls a route on the /settings API
    *
-   * @param  {string} path The setting route to call, eg `instance` or `context`
+   * @param  {string} id The setting id to call, eg `io.cozy.settings.instance` for `instance` route or `io.cozy.settings.context` for `context`route
    * @returns {object} The response from the route
    */
-  async get(path) {
+  async get(id) {
+    let path
+
+    if (id === 'io.cozy.settings.instance') {
+      path = 'instance'
+    } else if (id === 'io.cozy.settings.context') {
+      path = 'context'
```

Comment:
We've a lot of possible path, and not just `instance` or `context`. https://docs.cozy.io/en/cozy-stack/settings/ 

I'm wondering if we should not do something like: 

```
if(id.includes('io.cozy.settings.')){
path = substring(...)
} else {
// `Deprecated: in next versions of cozy-client, it will not be possible to query settings with an incomplete id

}
```